site/content/docs/development/reference.md

```diff
@@ -456,9 +458,227 @@ spec:
+`ClusterDeploymentTemplate`.
+
+`ClusterDeliveries` specify the type of configuration they accept through the `spec.selector` field. `Deliverable`s with matching
+`spec.selector` then create a logical delivery. This makes the values in the `Deliverable` available to all of the resources in the
+`ClusterDelivery`s `spec.resources`.
+
+```yaml
+apiVersion: carto.run/v1alpha1
+kind: ClusterDelivery
+metadata:
+  name: supplychain
+spec:
+
+  # specifies the label key-value pair to select deliverables. (required)
+  #
+  selector:
+    app.tanzu.vmware.com/deliverable-type: web---deliverable
+
+  # see specification of params in supply-chain
+  params: []
+
+  # set of resources that will take care of bringing the application to a
+  # deliverable state. (required, at least 1)
+  #
+  resources:
+    # name of the resource to be referenced by further resources in the chain.
+    # (required, unique)
+    #
+    - name: config-provider
+      # reference to a template object. (required)
+      #
+      templateRef:
+        kind: ClusterSourceTemplate
+        name: config-source
+
+    - name: additional-config-provider
+      templateRef:
+        kind: ClusterSourceTemplate
+        name: another-config-source
+
+    - name: deployer
+      templateRef:
+        kind: ClusterDeploymentTemplate
+        name: app-deploy
+      # a single resource that provides the location (url and revision) of
+      # configuration required for deployment,
+      # in a template, these can be consumed as:
+      #
+      #    $(deployment.url)$
+      #    $(deployment.revision)$
+      #
+      # (required)
+      deployment:
+        # name of the resource to provide the source information. (required)
+        #
+        resource: config-provider
+
+      # a set of resources that provide additional information for deployment located
+      # at the specified url and revision.
+      #
+      # in a template, these can be consumed as:
+      #
+      #    $(sources.<name>.url)$
+      #    $(sources.<name>.revision)$
+      #
+      # if there is only one source, it can be consumed as:
+      #
+      #    $(source.url)$
+      #    $(sources.revision)$
+      #
+      # (optional)
+      sources:
+        # name of the resource to provide the source information. (required)
+        #
+        - resource: additional-config-provider
+          # name to be referenced in the template via a query over the list of
+          # sources (for instance, `$(sources.provider.url)`.
+          #
+          # (required, unique in this list)
+          #
+          name: addtnl
+
+      # see specification for params in supply chain resources
+      params: []
+```
+_ref: [pkg/apis/v1alpha1/cluster_delivery.go](../../../pkg/apis/v1alpha1/cluster_delivery.go)_
+
+### ClusterDeploymentTemplate
+
+`ClusterDeploymentTemplate` indicates how the delivery should configure the environment (namespace/cluster).
```

Comment:
```suggestion
A `ClusterDeploymentTemplate` indicates how the delivery should configure the environment (namespace/cluster).
```
Just because it looks better with the &lt;pre&gt; block when rendered